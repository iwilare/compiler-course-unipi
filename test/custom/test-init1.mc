struct A {
    int a;
    int b;
}

int c = 3;

struct A s;

int main() {
    int a = 1;
    int b = 2;
    s.a = 10;
    s.b = 20;
    print(a);
    print(b);
    print(c);
    print(s.a);
    print(s.b);
    a++;
    b++;
    c++;
    s.a++;
    s.b++;
    print(a);
    print(b);
    print(c);
    print(s.a);
    print(s.b);
    return 0;
}
