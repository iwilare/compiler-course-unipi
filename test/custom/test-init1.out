1
2
3
10
20
2
3
4
11
21
